packages/yoshi-flow-legacy/src/commands/test.js

```diff
@@ -100,12 +100,14 @@ module.exports = runner.command(
         specsPattern.push(...globby.sync(globs.e2eTests));
         await bootstrapCdn();
       }
+      
+      const mochaTimeout = process.env.MOCHA_TIMEOUT || 30000;
```

Comment:
```suggestion
      const mochaTimeout = process.env.MOCHA_TIMEOUT || '30000';
```

That will probably work, but just to go on the safe/typed side.